src/Telegram.Bot/Converters/ChatActionConverter.cs

```diff
@@ -6,37 +6,40 @@ namespace Telegram.Bot.Converters;
 
 internal class ChatActionConverter : EnumConverter<ChatAction>
 {
-    static readonly IReadOnlyDictionary<string, ChatAction> StringToEnum =
```

Comment:
Recently we changed our codebase to the file scoped namespaces and it looks like we have an incorrect diff here.